cms/templates/home_page.html

```diff
@@ -9,6 +9,7 @@
 {% endblock %}
 
 {% block content %}
+<div class="cms-page">
```

Comment:
You can just add `cms-page` to the top level element instead of adding another wrapper element, right? `<div class="cms-page home-page">`

Reply:
That's the contrainer for one section of the product or home page so that's why had to add a parent contrainer which could aplly it for the whole page.

Earlier I had it besides the home-page so then it applies to the hero section of the page.